draft-claise-semver.md

```diff
@@ -147,7 +145,7 @@ RFC-formatted document as necessary.
 ~~~~~
 * a9d7d29 (tag:2.0.0, tag:RFCXXXX) Merge branch 'v2'
```

Comment:
There are a few things I don't understand about the example:

- The minor and patch versions seem to start at 0, but the major and beta versions seem to start at 1. Why?
- Why is the first major version "v1" but the second major version is just "2"? Where did the "v" come from?
- What is the meaning of a commit that has no version number tag associated with it? Is it that the changes made were not significant enough (e.g., just editorial) to warrant any of the four version numbers being incremented? This is probably worth some explanatory text above.